junit5/container/src/main/java/org/jboss/arquillian/junit5/container/JUnitJupiterDeploymentAppender.java

```diff
@@ -22,6 +24,7 @@ protected Archive<?> buildArchive() {
                 .addAsServiceProvider(
                         TestRunner.class,
                         JUnitJupiterTestRunner.class)
-                .addAsServiceProvider(TestEngine.class, JupiterTestEngine.class);
+                .addAsServiceProvider(TestEngine.class, JupiterTestEngine.class)
+                .addAsServiceProvider(Extension.class, ArquillianParameterResolver.class);
```

Comment:
This might be removed. This was an attempt to use the parameter resolver in the container. I'm not yet sure if this is needed yet or not.